### ISSUE TITLE
Error: dj.set('restore')  Parameter name does not exist

### ISSUE THREAD

**Issue by bertramr** [datajoint/datajoint-matlab#63]
```
>> dj.set('restore')
Error using dj.set (line 33)
Parameter name does not exist
````
dj.set.m, Line 33:
``` 
    assert(isfield(STATE,name), 'Parameter name does not exist')
```
'restore' is not a field of STATE

**Comment by dimitri-yatsenko** [comment]
indeed this option is described under USAGE but never implemented.  Will fix in new release.

**Comment by eywalker** [comment]
Fixed in #64
